### PR TITLE
Requesting review for bootstrap project

### PR DESCRIPTION
**Pull request by mkezfarias**
Hello, I completed the project newsweek using bootstrap and would like to request review please.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,867 @@
...
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/4f5233a2df.js" crossorigin="anonymous"></script>
+    <title>Newsweek - News, Analysis, Politics, Business, Technology</title>
+</head>
+
+<body>
+
+    <header class="nav-header">
+        <nav class="container-fluid">
+            <div class="py-xl-2 p-md-1 flex-nowrap d-flex flex-row justify-content-around align-items-baseline" style="background-color:red !important;">
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,867 @@
...
+                <div class="text-light d-none d-lg-block d-xl-block">
+                    <h6>Wed, Oct 23, 2019M</h6>
+                </div>
+                <img alt="newsweek logo" class="col-xl-4 col-lg-4 col-4 top-logo pt-lg-3" src="images/newsweek-logo.png">
+                <div class="btn-group">
+                    <button type="button" class="text-nowrap h6 font-weight-bold btn-primary" style="background-color:red !important;border:none;border-radius:0;">SIGN IN</button>
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,867 @@
...
+                    <button type="button" class="text-nowrap h6 text-danger font-weight-bold btn-light" style="background-color:white !important;border:none;border-radius:0;">SUBSCRIBE ></button>
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,867 @@
...
+                </div>
+                <span class="text-light lead d-lg-none d-xl-none" style="color:#fb9a94!important;font-size: 30px!important;"><i class="fas fa-bars"></i></span>
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,867 @@
...
+            </div>
+            <div><img alt="top" src="images/navredthing.PNG" class="float-right d-none d-xl-block d-lg-block"></div>
+        </nav>
+        <nav class="container-fluid d-none d-lg-block d-xl-block" style="width: 90vw;">
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,867 @@
...
+                <li class="nav-item h5 font-weight-bolder hhlink border-right py-0 my-2">
+                    <a class="nav-link active text-dark" href="#">Health</a>
+                </li>
+                <li class="nav-item h5 font-weight-bolder hhlink border-right py-0 my-2">
+                    <a class="nav-link active text-dark" href="#">Opinion</a>
+                </li>
+                <li class="nav-item h5 font-weight-bolder hhlink border-right py-0 my-2">
+                    <a class="nav-link active text-dark" href="#">Vantage</a>
+                </li>
+                <li class="nav-item h6 hhlink py-2 px-2 text-lg-left my-2 text-nowrap" style="color:gray">
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,867 @@
...
+                    <i class="fas fa-search px-2"></i>Search
+                </li>
+            </ul>
+        </nav>
+    </header>
+
+    <main class="container-fluid">
+
+        <div id="first-half" class="row mx-auto" style="position:relative;">
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,867 @@
...
+            <div id="left" class="row mx-4 mr-lg-2 col-12 col-lg-12 col-md-12 col-xl-8 p-4 px-md-0 mx-md-0 px-lg-0" style="box-sizing: border-box;">
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 9]
```diff
@@ -0,0 +1,867 @@
...
+                        </article>
+                    </div>
+
+                    <div id="more-stories" class="order-2 col-12 mt-2 mx-0 px-0 pt-2">
+                        <div class="pl-0 pl-0 block-title h6 text-secondary col-12 py-2 my-0">MORE STORIES</div>
+                        <article class="pl-0 ml-0 clearfix border border-left-0 border-right-0 py-3 border-bottom-0 border-secondary">
+                            <h3 class="d-none">featured article</h3>
+                            <img class="pl-0 ml-0 float-left img-fluid feat-img col-5" src="images/ms1.PNG" alt="featured story">
+                            <p class="h6 float left">Warren's Economic Plans Would Not Make Recession Likelier: Ex-Fed Chair </p>
+                            <p class="text-secondary card-text" style="line-height:14px!important;">The Massachusetts senator and Democratic 2020 presidential candidate is pledging to overhaul the U.S. economy with a number of reforms.</p>
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 10]
```diff
@@ -0,0 +1,867 @@
...
+                        </article>
+                        <article class="pl-0 ml-0 clearfix border border-left-0 border-right-0 py-3 border-bottom-0 border-secondary">
+                            <h3 class="d-none">featured article</h3>
+                            <img class="pl-0 ml-0 float-left img-fluid feat-img col-5" src="images/ms2.PNG" alt="featured story">
+                            <p class="h6 float left">Trump Handed Putin Major Victories for Nothing: Former Ambassador to Russia </p>
+                            <p class="text-secondary card-text" style="line-height:14px!important;">Michael McFaul said President Donald Trump has "personalized" and "privatized" U.S. foreign policy at great cost to the country.</p>
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 11]
```diff
@@ -0,0 +1,867 @@
...
+                        </article>
+                        <article class="pl-0 ml-0 clearfix border border-left-0 border-right-0 py-3 border-bottom-0 border-secondary">
+                            <h3 class="d-none">featured article</h3>
+                            <img class="pl-0 ml-0 float-left img-fluid feat-img col-5" src="images/ms3.PNG" alt="featured story">
+                            <p class="h6 float left">California Wildfire Map: Kincade, Waterman Canyon, Palisades Fire Locations</p>
+                            <p class="text-secondary card-text" style="line-height:14px!important;">Officials urged residents to remain alert and prepare to leave because high Santa Ana winds could cause the fires to spread rapidly.</p>
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 12]
```diff
@@ -0,0 +1,867 @@
...
+                        </article>
+                        <article class="pl-0 ml-0 clearfix border border-left-0 border-right-0 py-3 border-bottom-0 border-secondary">
+                            <h3 class="d-none">featured article</h3>
+                            <img class="pl-0 ml-0 float-left img-fluid feat-img col-5" src="images/ms4.PNG" alt="featured story">
+                            <p class="h6 float left">Giuliani's Ukraine Actions for Trump Are 'Highest of High Crimes': Law Professor </p>
+                            <p class="text-secondary card-text" style="line-height:14px!important;">The former New York City mayor was acting in a personal capacity on behalf of President Donald Trump while pursuing Ukraine conspiracy theories.</p>
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 13]
```diff
@@ -0,0 +1,867 @@
...
+                        </article>
+                        <article class="pl-0 ml-0 clearfix border border-left-0 border-right-0 py-3 border-bottom-0 border-secondary">
+                            <h3 class="d-none">featured article</h3>
+                            <img class="pl-0 ml-0 float-left img-fluid feat-img col-5" src="images/ms5.PNG" alt="featured story">
+                            <p class="h6 float left">Russia Shows Off New Syria Map, Sends Troops to Border After Turkey Deal</p>
+                            <p class="text-secondary card-text" style="line-height:14px!important;">The Russian Defense Ministry released a new map following the country's deal with Turkey to pause military operations and facilitate the withdrawal of Kurdish-led forces from northeastern Syria.</p>
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 14]
```diff
@@ -0,0 +1,867 @@
...
+                        </article>
+                        <article class="pl-0 ml-0 clearfix border border-left-0 border-right-0 py-3 border-bottom-0 border-secondary">
+                            <h3 class="d-none">featured article</h3>
+                            <img class="pl-0 ml-0 float-left img-fluid feat-img col-5" src="images/ms6.PNG" alt="featured story">
+                            <p class="h6 float left">Senator: U.S. Right to Withdraw Syria Troops If Turkey Planned Genocide </p>
+                            <p class="text-secondary card-text" style="line-height:14px!important;">The need to unplug is greater than ever—and there is no better way to do so than in the wild where there are no distractions, no vehicles and no cell phone signal.</p>
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 15]
```diff
@@ -0,0 +1,867 @@
...
+                        </article>
+                        <article class="pl-0 ml-0 clearfix border border-left-0 border-right-0 py-3 border-bottom-0 border-secondary">
+                            <h3 class="d-none">featured article</h3>
+                            <img class="pl-0 ml-0 float-left img-fluid feat-img col-5" src="images/ms7.PNG" alt="featured story">
+                            <p class="h6 float left">Trump's Mental Impairment Means He Can't Think Strategically: Psychiatrist</p>
+                            <p class="text-secondary card-text" style="line-height:14px!important;">The need to unplug is greater than ever—and there is no better way to do so than in the wild where there are no distractions, no vehicles and no cell phone signal.</p>
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 16]
```diff
@@ -0,0 +1,867 @@
...
+                        </article>
+                        <article class="pl-0 ml-0 clearfix border border-left-0 border-right-0 py-3 border-bottom-0 border-secondary">
+                            <h3 class="d-none">featured article</h3>
+                            <img class="pl-0 ml-0 float-left img-fluid feat-img col-5" src="images/ms8.PNG" alt="featured story">
+                            <p class="h6 float left">Babies Can Process Numbers Before They Can Count </p>
+                            <p class="text-secondary card-text" style="line-height:14px!important;">The need to unplug is greater than ever—and there is no better way to do so than in the wild where there are no distractions, no vehicles and no cell phone signal.</p>
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 17]
```diff
@@ -0,0 +1,867 @@
...
+                        </article>
+                        <article class="pl-0 ml-0 clearfix border border-left-0 border-right-0 py-3 border-bottom-0 border-secondary">
+                            <h3 class="d-none">featured article</h3>
+                            <img class="pl-0 ml-0 float-left img-fluid feat-img col-5" src="images/ms9.PNG" alt="featured story">
+                            <p class="h6 float left">Hidden Tracks: Stunning Trails Around the Globe </p>
+                            <p class="text-secondary card-text" style="line-height:14px!important;">The need to unplug is greater than ever—and there is no better way to do so than in the wild where there are no distractions, no vehicles and no cell phone signal.</p>
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 18]
```diff
@@ -0,0 +1,867 @@
...
+                        </article>
+                        <article class="pl-0 ml-0 clearfix border border-left-0 border-right-0 py-3 border-secondary">
+                            <h3 class="d-none">featured article</h3>
+                            <img class="pl-0 ml-0 float-left img-fluid feat-img col-5" src="images/ms10.PNG" alt="featured story">
+                            <p class="h6 float left">Babies Can Process Numbers Before They Can Count </p>
+                            <p class="text-secondary card-text" style="line-height:14px!important;">The need to unplug is greater than ever—and there is no better way to do so than in the wild where there are no distractions, no vehicles and no cell phone signal.</p>
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 19]
```diff
@@ -0,0 +1,867 @@
...
+                            <h4 class="card-title h5">What Can a Degree in Public Health Do For You?</h4>
+                            <p class="card-text">The field of public health is a powerful draw for those who feel inspired to make a positive difference in their communities. Find out more.</p>
+                        </div>
+                    </article>
+                </div>
+            </div>
+        </div>
+    </main>
+
+    <div id="bottom-half" class="row m-auto" style="width: 90%!important;">
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 20]
```diff
@@ -0,0 +1,867 @@
...
+                <h4 class="card-title h6 font-weight-bolder"><a class="text-dark" href="#">A Pence Presidency Is No Longer Just Dinner Party Chatter—Is He Up to It?</a></h4>
+            </article>
+            <article class="py-0">
+                <img alt="magazzine1" class="mx-0 px-0 col-12 card-img-top img-fluid feat-img" src="images/m4.PNG">
+                <h4 class="card-title h6 font-weight-bolder"><a class="text-dark" href="#">A Pence Presidency Is No Longer Just Dinner Party Chatter—Is He Up to It?</a></h4>
+            </article>
+        </div>
+
+        <div class="3rd-col col-xl-3 col-lg-3 col-md-3 col-11">
+            <img alt="magazzine" class="col-12 mx-auto mt-2 order-sm-0 order-xs-0" style="width: 200px;" src="images/inthemagazzine.PNG">
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 21]
```diff
@@ -0,0 +1,867 @@
...
+            </article>
+            <article class="py-0">
+                <img alt="magazzine1" class="mx-0 px-0 col-12 card-img-top img-fluid feat-img" src="images/m8.PNG">
+                <h4 class="card-title h6 font-weight-bolder"><a class="text-dark" href="#">A Pence Presidency Is No Longer Just Dinner Party Chatter—Is He Up to It?</a></h4>
+            </article>
+        </div>
+    </div>
+
+
+    <div id="editors-pick" class="row m-auto mt-4 pt-4 pb-4" style="width:90% !important;border-top:1px solid lightgray;">
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 22]
```diff
@@ -0,0 +1,867 @@
...
+        <div class="col-12">
+            <img alt="editors" class="mx-auto mt-2" style="position:absolute;left:50vw;" src="images/editors.PNG">
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 23]
```diff
@@ -0,0 +1,867 @@
...
+        </div>
+
+
+        <section class="row">
+            <h5 class="d-none">Pence presidency</h5>
+            <div class="1st-col col-xl-3 col-lg-3 col-12 col-md-6 pt-4">
+                <article class="py-4">
+                    <img alt="magazzine1" class="mx-0 px-0 col-12 card-img-top img-fluid feat-img" src="images/e1.PNG">
+                    <h4 class="card-title h6 font-weight-bolder"><a class="text-dark" href="#">A Pence Presidency Is No Longer Just Dinner Party Chatter—Is He Up to It?</a></h4>
+                    <p class="text-secondary card-text" style="line-height:14px!important;">The need to unplug is greater than ever—and there is no better way to do so than in the wild where there are no distractions, no vehicles and no cell phone signal.</p>
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 24]
```diff
@@ -0,0 +1,867 @@
...
+
+                </article>
+                <article class="py-0">
+                    <img alt="magazzine1" class="mt-4 mx-0 px-0 col-12 card-img-top img-fluid feat-img" src="images/e2.PNG">
+                    <h4 class="card-title h6 font-weight-bolder"><a class="text-dark" href="#">A Pence Presidency Is No Longer Just Dinner Party Chatter—Is He Up to It?</a></h4>
+                    <p class="text-secondary card-text" style="line-height:14px!important;">The need to unplug is greater than ever—and there is no better way to do so than in the wild where there are no distractions, no vehicles and no cell phone signal.</p>
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 25]
```diff
@@ -0,0 +1,867 @@
...
+                </article>
+            </div>
+
+            <div class="2nd-col col-xl-3 col-lg-3 col-12 col-md-6">
+                <article class="py-5">
+                    <img alt="magazzine1" class="mx-0 px-0 col-12 card-img-top img-fluid feat-img" src="images/e3.PNG">
+                    <h4 class="card-title h6 font-weight-bolder"><a class="text-dark" href="#">A Pence Presidency Is No Longer Just Dinner Party Chatter—Is He Up to It?</a></h4>
+                    <p class="text-secondary card-text" style="line-height:14px!important;">The need to unplug is greater than ever—and there is no better way to do so than in the wild where there are no distractions, no vehicles and no cell phone signal.</p>
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 26]
```diff
@@ -0,0 +1,867 @@
...
+                </article>
+                <article class="py-0">
+                    <img alt="magazzine1" class="mx-0 px-0 col-12 card-img-top img-fluid feat-img" src="images/e4.PNG">
+                    <h4 class="card-title h6 font-weight-bolder"><a class="text-dark" href="#">A Pence Presidency Is No Longer Just Dinner Party Chatter—Is He Up to It?</a></h4>
+                    <p class="text-secondary card-text" style="line-height:14px!important;">The need to unplug is greater than ever—and there is no better way to do so than in the wild where there are no distractions, no vehicles and no cell phone signal.</p>
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 27]
```diff
@@ -0,0 +1,867 @@
...
+                </article>
+            </div>
+
+            <div class="3rd-col col-xl-3 col-lg-3 col-12 col-md-6">
+                <article class="py-5">
+                    <img alt="magazzine1" class="mx-0 px-0 col-12 card-img-top img-fluid feat-img" src="images/e5.PNG">
+                    <h4 class="card-title h6 font-weight-bolder"><a class="text-dark" href="#">A Pence Presidency Is No Longer Just Dinner Party Chatter—Is He Up to It?</a></h4>
+                    <p class="text-secondary card-text" style="line-height:14px!important;">The need to unplug is greater than ever—and there is no better way to do so than in the wild where there are no distractions, no vehicles and no cell phone signal.</p>
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 28]
```diff
@@ -0,0 +1,867 @@
...
+                </article>
+                <article class="py-0">
+                    <img alt="magazzine1" class=" mx-0 px-0 col-12 card-img-top img-fluid feat-img" src="images/e6.PNG">
+                    <h4 class="card-title h6 font-weight-bolder"><a class="text-dark" href="#">A Pence Presidency Is No Longer Just Dinner Party Chatter—Is He Up to It?</a></h4>
+                    <p class="text-secondary card-text" style="line-height:14px!important;">The need to unplug is greater than ever—and there is no better way to do so than in the wild where there are no distractions, no vehicles and no cell phone signal.</p>
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 29]
```diff
@@ -0,0 +1,867 @@
...
+                </article>
+            </div>
+
+            <div class="4th-col col-xl-3 col-lg-3 col-12 col-md-6">
+                <article class="py-5">
+                    <img alt="magazzine1" class="mx-0 px-0 col-12 card-img-top img-fluid feat-img" src="images/e7.PNG">
+                    <h4 class="card-title h6 font-weight-bolder"><a class="text-dark" href="#">A Pence Presidency Is No Longer Just Dinner Party Chatter—Is He Up to It?</a></h4>
+                    <p class="text-secondary card-text" style="line-height:14px!important;">The need to unplug is greater than ever—and there is no better way to do so than in the wild where there are no distractions, no vehicles and no cell phone signal.</p>
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 30]
```diff
@@ -0,0 +1,867 @@
...
+                </article>
+                <article class="py-0">
+                    <img alt="magazzine1" class="mx-0 px-0  col-12 card-img-top img-fluid feat-img" src="images/e8.PNG">
+                    <h4 class="card-title h6 font-weight-bolder"><a class="text-dark" href="#">A Pence Presidency Is No Longer Just Dinner Party Chatter—Is He Up to It?</a></h4>
+                    <p class="text-secondary card-text" style="line-height:14px!important;">The need to unplug is greater than ever—and there is no better way to do so than in the wild where there are no distractions, no vehicles and no cell phone signal.</p>
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 31]
```diff
@@ -0,0 +1,867 @@
...
+                </article>
+            </div>
+        </section>
+    </div>
+
+
+    <div id="slideshow" class="row m-auto mt-3 py-3" style="width:90% !important;border-top:1px solid lightgray;">
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 32]
```diff
@@ -0,0 +1,867 @@
...
+        <div class="col-12">
+            <img alt="slideshow" class="mx-auto mt-2" style="position:absolute;left:40vw;" src="images/featuredslide.PNG">
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 33]
```diff
@@ -0,0 +1,867 @@
...
+                <article class="py-2">
+                    <img alt="magazzine1" class=" mx-0 px-0 col-12 card-img-top img-fluid feat-img" src="images/fs6.PNG">
+                    <h4 class="card-title h6 font-weight-bolder"><a class="text-dark" href="#">A Pence Presidency Is No Longer Just Dinner Party Chatter—Is He Up to It?</a></h4>
+                </article>
+            </div>
+
+        </section>
+    </div>
+
+    <div id="us" class="row m-auto mt-3 py-3" style="width:90% !important;border-top:1px solid lightgray;">
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 34]
```diff
@@ -0,0 +1,867 @@
...
+                <div class="row pt-1">
+                    <img alt="magazzine1" class="mx-0 px-0 col-4 card-img-top img-fluid feat-img" src="images/us5.PNG">
+                    <h4 class="col-8 h6 font-weight-bolder"><a class="text-dark" href="#">Trump Threatens to Sue Democrats and 'Shifty Adam Schiff' for Fraud</a></h4>
+                </div>
+            </div>
+
+        </section>
+    </div>
+
+    <div id="world" class="row m-auto mt-3 py-3" style="width:90% !important;border-top:1px solid lightgray;">
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 35]
```diff
@@ -0,0 +1,867 @@
...
+                    <img alt="magazzine1" class="mx-0 px-0 col-4 card-img-top img-fluid feat-img" src="images/w4.PNG">
+                    <h4 class="col-7 h6 font-weight-bolder"><a class="text-dark" href="#">
+                            Woman Fakes Pregnancy to Try and Avoid $40 Airline Excess Baggage Fees</a></h4>
+                </div>
+            </div>
+
+        </section>
+    </div>
+
+    <div id="b2" class="row m-auto mt-3 py-3" style="width:90% !important;border-top:1px solid lightgray;">
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 36]
```diff
@@ -0,0 +1,867 @@
...
+                    <img alt="magazzine1" class="mx-0 px-0 col-4 card-img-top img-fluid feat-img" src="images/b4.PNG">
+                    <h4 class="col-7 h6 font-weight-bolder"><a class="text-dark" href="#">
+                                Woman Fakes Pregnancy to Try and Avoid $40 Airline Excess Baggage Fees</a></h4>
+                </div>
+            </div>
+
+        </section>
+    </div>
+
+    <div id="tech" class="row m-auto mt-3 py-3" style="width:90% !important;border-top:1px solid lightgray;">
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 37]
```diff
@@ -0,0 +1,867 @@
...
+                                    Woman Fakes Pregnancy to Try and Avoid $40 Airline Excess Baggage Fees</a></h4>
+                </div>
+            </div>
+
+
+        </section>
+    </div>
+
+
+    <div id="culture" class="row m-auto mt-3 py-3" style="width:90% !important;border-top:1px solid lightgray;">
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 38]
```diff
@@ -0,0 +1,867 @@
...
+                    <h4 class="col-7 h6 font-weight-bolder"><a class="text-dark" href="#">
+                                        Woman Fakes Pregnancy to Try and Avoid $40 Airline Excess Baggage Fees</a></h4>
+                </div>
+            </div>
+
+
+        </section>
+    </div>
+
+    <div id="sports" class="row m-auto mt-3 py-3" style="width:90% !important;border-top:1px solid lightgray;">
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 39]
```diff
@@ -0,0 +1,867 @@
...
+                <div class="row pt-1">
+                    <img alt="magazzine1" class="mx-0 px-0 col-4 card-img-top img-fluid feat-img" src="images/spo4.PNG">
+                    <h4 class="col-7 h6 font-weight-bolder"><a class="text-dark" href="#">
+                                            Woman Fakes Pregnancy to Try and Avoid $40 Airline Excess Baggage Fees</a></h4>
+                </div>
+            </div>
+        </section>
+    </div>
+
+    <div class="row border-top pt-4" style="width: 100% !important;">
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 40]
```diff
@@ -0,0 +1,867 @@
...
+            <p class="h5 text-danger font-weight-bolder col-2">SUBSCRIBE</p>
+            <img alt="subscribe2" class="col-12" src="images/subscribe1.PNG">
+        </div>
+        <div class="mx-auto col-lg-4 col-xl-4 col-12">
+            <p class="h5 text-danger font-weight-bolder col-2">NEWSLETTER</p>
+            <img alt="subscribe2" class="col-12" src="images/nesletter2.PNG">
+        </div>
+    </div>
+
+    <footer class="footer row border-top py-4" style="width: 100% !important;">
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: index.html [hunk 41]
```diff
@@ -0,0 +1,867 @@
...
+                        </li>
+                    </ul>
+                </nav>
+            </div>
+
+            <div class="footerbtm col-12 col-md-5 col-lg-5 col-xl-5">
+                <nav>
+                    <ul class="nav">
+                        <li class="nav-item">
+                            <a class="nav-link flin" style="font-weight: bolder;" href="#">Editions:</a>
```
Comment (suretrust) [inline index.html]: Let's take this inline styling to the CSS file.

File: styles.css [hunk 42]
```diff
@@ -0,0 +1,50 @@
+nav {
+  margin: 0 !important;
+  padding: 0 !important;
+}
```
Comment (suretrust) [inline styles.css]: You could do without the `!important` flag in some of your code.

File: styles.css [hunk 43]
```diff
@@ -0,0 +1,50 @@
...
+
+li.hhlink :hover {
+  color: red !important;
```
Comment (suretrust) [inline styles.css]: You could do without the `!important` flag in some of your code.

File: styles.css [hunk 44]
```diff
@@ -0,0 +1,50 @@
...
+.card-body {
+  padding: 1rem 0 !important;
+}
+
+.mainmenu > * {
+  font-size: 1.4vw;
+}
+
+@media (min-width: 1008px) {
+  .ststst {
```
Comment (suretrust) [inline styles.css]: [Optional] The class naming could be better.

File: styles.css [hunk 45]
```diff
@@ -0,0 +1,50 @@
...
+    position: sticky !important;
```
Comment (suretrust) [inline styles.css]: I doubt you need `!important` here at all.

File: styles.css [hunk 46]
```diff
@@ -0,0 +1,50 @@
...
+    top: 50px;
+  }
+}
+
+.text-danger {
+  color: red !important;
```
Comment (suretrust) [inline styles.css]: If you haven't predefined a color before, there is no need for `!important` here.

File: styles.css [hunk 47]
```diff
@@ -0,0 +1,50 @@
...
+}
+
+.flin {
+  color: gray !important;
```
Comment (suretrust) [inline styles.css]: You could do without the `!important` flag in some of your code.

File: styles.css [hunk 48]
```diff
@@ -0,0 +1,50 @@
...
+.card-text {
+  font-size: 0.9rem;
+}
+
+.bg-light {
+  background-color: #f4f5f7 !important;
+}
+
+.card {
+  border: none !important;
```
Comment (suretrust) [inline styles.css]: You could do without the `!important` flag in some of your code.

File: styles.css [hunk 49]
```diff
@@ -0,0 +1,50 @@
...
+.feat-img {
+  width: 100%;
+}
+
+.card-text {
+  font-size: 0.9rem;
+}
+
+.bg-light {
+  background-color: #f4f5f7 !important;
```
Comment (suretrust) [inline styles.css]: You could do without the `!important` flag in some of your code.

**Review by suretrust — CHANGES_REQUESTED** [review]
Hi @mkezmkez,

**Great job thus far!** :raised_hands: Just a few suggestions to wrap it up:

- [x] Please see my comments on your code below. 

`[Explanation]` We don't use inline styling because it is against maintainability, reusability, scalability, and semantic markup (A way of writing and structuring your HTML such that it reinforces the semantics, or meaning, of the content rather than its appearance). You can also read [How to apply `!important` in CSS](https://www.geeksforgeeks.org/how-to-apply-important-in-css/) to fully understand my comments on the `!important` flag.

- [x] The navigation bar's position should be fixed on scroll.

- [x] On medium breakpoint, the `Opinion` section's width should be the width of the screen. Please refer to the images below:

opinion
opinion2

- [x] The `Latest News` section also has the issue highlighted above.

- [x] On medium screen, the `Sponsored Insight` and `Newsletter sign up` sections should have a width of the screen and aligned center. Please refer to the images below:

leaders
leaders2


- [ ] On medium screen, the `Editor's Picks` section should have 2 rows. Please refer to the images below:

editor
editor2

**Kindly submit another code review once these issues have been fixed.**

_Happy coding! :tada:_

File: styles.css (hunk 46 above)
Comment (mkezfarias) [reply styles.css]: There are pre-defined colors in bootstrap such as text-danger, that's why, I used important as necessary

File: styles.css (hunk 47 above)
Comment (mkezfarias) [reply styles.css]: There are pre-defined colors in bootstrap, that's why I used important as necessary

File: styles.css (hunk 48 above)
Comment (mkezfarias) [reply styles.css]: There are pre-defined card styles in bootstrap, that's why I used important as necessary

File: styles.css (hunk 49 above)
Comment (mkezfarias) [reply styles.css]: There are pre-defined colors in bootstrap, that's why I used important as necessary

**Comment by mkezfarias** [comment]
Resolved Issues

**Comment by ebukaume** [comment]
## Changes Required! :stop_sign: 

Hi @mkezmkez :wave: 

Nice job implementing some of the requested changes. Please, let's also get rid of the `!important` overuse.

You can read about why it is such a bad practice [here](https://speckyboy.com/good-bad-css-practices). Also, read more about specificity [here](https://www.w3schools.com/css/css_specificity.asp) and [here](https://dev.to/emmawedekind/css-specificity-1kca). 

Finally, instead of using the same class name as bootstrap, why not make your own class and apply it to the element and the style would be cascaded accordingly.

**Please submit another code review request after getting rid of those `!important` directives.**

Happy Coding :v: 
**Ebuka Umeokonkwo**
 [Github](https://github.com/ebukaume) | [Linkedin](https://www.linkedin.com/in/ebukaume) | [Slack](https://app.slack.com/team/UFDKXE68L) | [Twitter](https://twitter.com/ebukaume)

**Comment by ebukaume** [comment]
## Approved! :champagne: 

**Congratulations! you can go ahead and merge this branch.**

Happy Coding :v: 
**Ebuka Umeokonkwo**
 [Github](https://github.com/ebukaume) | [Linkedin](https://www.linkedin.com/in/ebukaume) | [Slack](https://app.slack.com/team/UFDKXE68L) | [Twitter](https://twitter.com/ebukaume)

**Pull request merged**